cli/src/main.rs

```diff
@@ -61,6 +64,9 @@ async fn main() -> Result<()> {
 
     LiquidSdk::init_logging(&data_dir_str, None)?;
 
+    let data_sync_url = std::env::var_os("SYNC_SERVICE_URL")
```

Comment:
I think it makes sense to make the SYNC_SERVICE_URL env variable optional as the default config defines it.

Reply:
It is optional, and always setting it to none if not set.

Reply:
Sorry, I read the code wrongly.